src/main/java/appeng/block/misc/BlockTinyTNT.java

```diff
@@ -54,15 +58,24 @@
 	public BlockTinyTNT()
 	{
 		super( Material.TNT );
-		this.setLightOpacity( 1 );
+
 		this.boundingBox = new AxisAlignedBB( 0.25f, 0.0f, 0.25f, 0.75f, 0.5f, 0.75f );
-		this.setFullSize( this.setOpaque( false ) );
+
+		this.setLightOpacity( 2 );
+		this.setFullSize( false );
+		this.setOpaque( false );
+
 		this.setSoundType( SoundType.GROUND );
 		this.setHardness( 0F );
 
-		// TODO: 1.11
-		// EntityRegistry.registerModEntity( EntityTinyTNTPrimed.class, "EntityTinyTNTPrimed", EntityIds.get(
-		// EntityTinyTNTPrimed.class ), AppEng.instance(), 16, 4, true );
+		EntityRegistry.registerModEntity( new ResourceLocation( AppEng.MOD_ID, EntityTinyTNTPrimed.class.getName() ), EntityTinyTNTPrimed.class,
+				"EntityTinyTNTPrimed", EntityIds.get( EntityTinyTNTPrimed.class ), AppEng.instance(), 16, 4, true );
+	}
+
+	@Override
+	public boolean isFullCube( IBlockState state )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Don't override a deprecated method or explicitly mark it as "@Deprecated". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)